frontend/src/component/admin/banners/BannersTable/BannersTable.tsx

```diff
@@ -0,0 +1,250 @@
+            header={
+                <PageHeader
+                    title={`Banners (${rows.length})`}
+                    actions={
+                        <>
+                            <ConditionallyRender
+                                condition={!isSmallScreen}
+                                show={
+                                    <>
+                                        <Search
+                                            initialValue={searchValue}
+                                            onChange={setSearchValue}
+                                        />
+                                        <PageHeader.Divider />
+                                    </>
+                                }
+                            />
+                            <Button
+                                variant='contained'
+                                color='primary'
+                                onClick={() => {
+                                    setSelectedBanner(undefined);
+                                    setModalOpen(true);
+                                }}
+                            >
+                                New banner
+                            </Button>
+                        </>
+                    }
+                >
+                    <ConditionallyRender
+                        condition={isSmallScreen}
+                        show={
+                            <Search
+                                initialValue={searchValue}
+                                onChange={setSearchValue}
+                            />
+                        }
+                    />
+                </PageHeader>
+            }
+        >
+            <SearchHighlightProvider value={getSearchText(searchValue)}>
+                <VirtualizedTable
+                    rows={rows}
+                    headerGroups={headerGroups}
+                    prepareRow={prepareRow}
+                />
+            </SearchHighlightProvider>
+            <ConditionallyRender
+                condition={rows.length === 0}
+                show={
+                    <ConditionallyRender
+                        condition={searchValue?.length > 0}
+                        show={
+                            <TablePlaceholder>
+                                No banners found matching &ldquo;
+                                {searchValue}
+                                &rdquo;
+                            </TablePlaceholder>
+                        }
+                        elseShow={
+                            <TablePlaceholder>
+                                No banners available. Get started by adding one.
+                            </TablePlaceholder>
+                        }
+                    />
+                }
+            />
+            {/* <BannerModal
+                banner={selectedBanner}
+                open={modalOpen}
+                setOpen={setModalOpen}
+            /> */}
```

Comment:
Probably it's fine, just making sure this does not stay forever ;)

Reply:
It won't, it will be introduced in the next PR. It's mostly a way of stopping me from doing everything in one go 😅 
Maybe next time I should simply remove it entirely.

Reply:
No worries, I imagined it was going to be included in next PR, if that's the case it's fine :+1: 

I usually forget to remove some comments from time to time and I just wanted to help you in case it was one of those cases ;)